erda.yml

```diff
@@ -249,6 +249,9 @@ services:
       - port: 10000
         protocol: "TCP"
         l4_protocol: "TCP"
+      - port: 8080
```

Comment:
Does image provider have invoker?

And you should add a global env `DICEHUB_ADDR` point to original addr for discover random-select issue.